builtin/gc.c

```diff
@@ -1858,6 +1858,7 @@ static int launchctl_schedule_plist(const char *exec_path, enum schedule_priorit
 		   "<string>%s/git</string>\n"
 		   "<string>--exec-path=%s</string>\n"
 		   "<string>for-each-repo</string>\n"
+		   "<string>--keep-going</string>\n"
```

Comment:
You might also want to update the example in Documentation/git-maintenance.txt accordingly.

Reply:
Well, it says "something like", and I considered it too much clutter ;-)